spec/istore/migration_spec.rb

```diff
@@ -1,18 +1,20 @@
 require 'spec_helper'
-describe "isagg" do
 
+describe "istore migrations" do
   it 'should be upgradeable' do
-    query("CREATE EXTENSION istore VERSION '0.1.3'")
-    query("ALTER EXTENSION istore UPDATE TO '0.1.4'")
-    query("ALTER EXTENSION istore UPDATE TO '0.1.3'")
```

Comment:
I'd leave theses upgrade function to still test old upgrade paths

Reply:
For some reason, I thought it' supposed to test the most recent path. Will revert this.